websites/P/Prydwen Institute/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.5",
+  "author": {
+    "name": "Kyrie",
+    "id": "368399721494216706"
+  },
+  "contributors": [
+    {
+      "name": "Sharkie",
+      "id": "400618023075184650"
+    },
+    {
+      "name": "Antillar",
+      "id": "277496099315515392"
+    }
+  ],
```

Comment:
Have they contributed at least 25% to the codebase? 🤔 
![image](https://user-images.githubusercontent.com/64903135/146614614-94e9c56e-43b0-4002-8969-cefe87fd7f2f.png)

Reply:
Well the both of them provided ALL the Art Assets for the presence and one of them is the site owner whom made the final decision on what to show up on `largeImage` and `details`
If that counts as contributing to every `Image` and `details` line then I guess so

Reply:
As the presence author you should own the art assets on your Discord Developer account. Furthermore, neither of those reasons would qualify for presence contributor

Reply:
Got it. Will omit ASAP.